### PR TITLE
Kapacitor 1.5.8

### PR DESCRIPTION
**Pull request by kelseiv**
- Kapacitor 1.5.8 release notes and feature docs
@sanderson will you take a look at this commit: https://github.com/influxdata/docs-v2/pull/2032/commits/89441b1326074bef2bb79c66a205f995d4ef67be - thank you!

### CODE REVIEW

File: content/kapacitor/v1.5/event_handlers/bigpanda.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+            .id('cpu_usage_check')
+            .message('Hey {{ index .Tags "host"}} / {{ .ID }}: is high!')
+            .details('https://example.com/dashboard/{{ index .Tags "host"}}')
+            .info(lambda: "total_used" > 70)
+            .warn(lambda: "total_used" > 80)
+            .crit(lambda: "total_used" > 90)
+            .stateChangesOnly()
+      .bigPanda()
+      .appKey('...')
+```
```
Comment (sanderson) [inline content/kapacitor/v1.5/event_handlers/bigpanda.md:110]:
~~~suggestion
```js
stream
  |from()
    .database('telegraf')
    .retentionPolicy('autogen')
    .measurement('cpu')
    .groupBy('host')
    .where(lambda: "cpu" == 'cpu-total')
  |eval(lambda: 100.0 - "usage_idle").as('total_used')
  |window().period(10s).every(10s)
  |mean('total_used').as('total_used')
  |alert()
    .id('cpu_usage_check')
    .message('Hey {{ index .Tags "host"}} / {{ .ID }}: is high!')
    .details('https://example.com/dashboard/{{ index .Tags "host"}}')
    .info(lambda: "total_used" > 70)
    .warn(lambda: "total_used" > 80)
    .crit(lambda: "total_used" > 90)
    .stateChangesOnly()
    .bigPanda()
      .appKey('...')
```
~~~

File: content/kapacitor/v1.5/event_handlers/bigpanda.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+Set to `true` to enable the BigPanda event handler.
+
+#### `token`
+
+Authorization Bearer token for BigPanda REST API.  
+
+#### `app-key`
+
+BigPanda integration App Key. To get your App Key, log in to your BigPanda account and select **Integrations** in the header toolbar, and then click the **New Integration**.
+Select **Alerts REST API**, click **Integrate** button, and then **Create an App Key**.
```
Comment (sanderson) [inline content/kapacitor/v1.5/event_handlers/bigpanda.md:41]:
```suggestion
BigPanda integration App Key. To get your App Key, log in to your BigPanda account and select **Integrations** in the header toolbar, and then click **New Integration**.
Select **Alerts REST API**, click **Integrate**, and then **Create an App Key**.
```

File: content/kapacitor/v1.5/event_handlers/bigpanda.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+ 
+#### `url`
+
+BigPanda Alert API url.
```
Comment (sanderson) [inline content/kapacitor/v1.5/event_handlers/bigpanda.md:45]: Does this have a default?

File: content/kapacitor/v1.5/event_handlers/bigpanda.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+See [Alert Correlation Logic](https://docs.bigpanda.io/docs/alert-correlation-logic) for more information.
+
+By default, the handler maps the Kapacitor values below to the BigPanda Alert or Event fields as follows:
+
+| Value           | BigPanda Alert Field       |
+| ----            | ----                       |
+| alert ID        | check                      |
+| message         | description                |
+| details         | details                    |
+| alert task name | task                       |
```
Comment (sanderson) [inline content/kapacitor/v1.5/event_handlers/bigpanda.md:70]:
```suggestion
By default, the handler maps the Kapacitor task and alert properties below to the BigPanda Alert or Event fields as follows:

| Value           | BigPanda Alert Field       |
| ----            | ----                       |
| `id`            | check                      |
| `message`       | description                |
| `details`       | details                    |
| `TaskName`      | task                       | 
```

File: content/kapacitor/v1.5/event_handlers/bigpanda.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+| Value           | BigPanda Alert Field       |
+| ----            | ----                       |
+| alert ID        | check                      |
+| message         | description                |
+| details         | details                    |
+| alert task name | task                       | 
+
+All EventData tags and fields are appended into BigPanda Alert as **Additional attributes**. See 
+[BigPanda Alert REST API](https://docs.bigpanda.io/reference#alerts) for more information.
```
Comment (sanderson) [inline content/kapacitor/v1.5/event_handlers/bigpanda.md:73]:
```suggestion
All EventData tags and fields are appended to the BigPanda Alert as **Additional attributes**.
For more information, see [BigPanda Alert REST API](https://docs.bigpanda.io/reference#alerts). 
```

**Review by sanderson — APPROVED** [review]
Some minor suggestions.

File: content/kapacitor/v1.5/event_handlers/bigpanda.md (hunk 3 above)
Comment (kelseiv) [reply content/kapacitor/v1.5/event_handlers/bigpanda.md:45]: https://api.bigpanda.io/data/v2/alerts"

**Review by psteinbachs — APPROVED** [review]
+1 This looks solid to me.

**Pull request merged**